controller/destination/server.go

```diff
@@ -126,6 +132,29 @@ func (s *server) Get(dest *common.Destination, stream pb.Destination_GetServer)
 	return err
 }
 
+func (s *server) echoIPDestination(host string, port int, stream pb.Destination_GetServer) error {
+	ip, err := util.ParseIPV4(host)
+	if err != nil {
+		return err
+	}
+	addrs := make([]common.TcpAddress, 1)
```

Comment:
consider stack allocation when you know the size ahead of time:
```go
addrs := []common.TcpAddress{
	{
		Ip:   ip,
		Port: uint32(port),
	},
}
```

from https://segment.com/blog/allocation-efficiency-in-high-performance-go-services/:
```
Go prefers allocation on the stack
```